src/fixedpoint.test.ts

```diff
@@ -29,6 +29,11 @@ describe("fpnum", () => {
       fixedInt.val = 2n;
       expect(fixedInt.val).toEqual(2n);
     });
+
+    test("converts to float correctly", () => {
+      const fixedInt = new FixedInt(100n, 2);
+      expect(fixedInt.toFloat()).toEqual(1.0);
+    });
```

Comment:
This is great! 

Parameterized specs here would be useful. Its a nice way to test multiple combos of inputs an expected outputs using the same code.

Example:

https://github.com/peeldao/fpnum/pull/6/files#diff-641ae9a74bc2f9e196ff7b818a52f458f29aa61a72f606c11aa5acbcf0918b74R5-R14

Reply:
Nicee good to know, thanks! I've implemented.